src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -327,4 +285,69 @@ function renderGuidance(
   return returnElements;
 }
 
+interface RenderProps {
```

Comment:
Sorry to add more comments, but we should rename this to match the style guide: https://github.com/mcode/pathways/wiki/Code-Style-Guide#props

```suggestion
interface ExpandedNodeRenderProps {
```